### ISSUE TITLE
PTE Remapping 32 bit

### ISSUE THREAD

**Issue by hellotheregeneralkenobi1** [google/rekall#482]
I am wondering why there is no implementation for PTE Remapping in 32 bit.
Is there a reason for it? 

Thanks in advance :+1:

**Comment by scudette** [comment]
No reason other than that no one really runs 32 bit systems any more so the motivation is pretty limited.  The 32 bit version supports traditional methods of acquisition so they do work usually (at least as well as other acquisition tools).

**Comment by hellotheregeneralkenobi1** [comment]
> No reason other than that no one really runs 32 bit systems any more so the motivation is pretty limited. The 32 bit version supports traditional methods of acquisition so they do work usually (at least as well as other acquisition tools).

Yes you are correct, however hooking such "acquisition" functions (MmMapIoSpace and MapViewOfSection) is probably more common in 32 bit which means that the acquisition won't be too much reliable. 
Also, legacy support for old computers is needed most of the time...
Thanks for the quick answer
